src/wallet/asyncrpcoperation_saplingmigration.cpp

```diff
@@ -191,7 +191,9 @@ CAmount AsyncRPCOperation_saplingmigration::chooseAmount(const CAmount& availabl
     return amount;
 }
 
-// Unless otherwise specified, the migration destination address is the address for Sapling account 0
+// Unless otherwise specified, the migration destination address is the address
+// the legacy Sapling account (0x7FFFFFFF) at the smallest diversifier index
+// that produces a valid diversified address.
```

Comment:
```suggestion
// Unless otherwise specified, the migration destination address is the
// default address for the key at m/32'/coin_type'/0x7FFFFFFF'/0'
```